index.js

```diff
@@ -1,3 +1,9 @@
 (function (){
     console.log("this function works")
+    console.log("this function works")
```

Comment:
why did you add 2 many console.log messages

Reply:
hello

Reply:
hello

Reply:
Hi from Itzik

Reply:
hello

Reply:
hello

Reply:
hello

Reply:
hi

Reply:
ok ok

Reply:
ok

Reply:
ok